src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -50,21 +51,42 @@ private Solution CreateSolutionWithProjectAndDocuments()
             var projectId = ProjectId.CreateNewId();
 
             return CreateSolution()
-                .AddProject(projectId, "goo", "goo.dll", LanguageNames.CSharp)
+                .AddProject(projectId, "proj1", "proj1.dll", LanguageNames.CSharp)
```

Comment:
Why?

Reply:
I needed to add more projects in some tests so it makes sense to number them.